lib/matplotlib/pylab.py

```diff
@@ -240,21 +236,7 @@
 # bring all the symbols in so folks can import them from
 # pylab in one fell swoop
 
-## We are still importing too many things from mlab; more cleanup is needed.
-
-from matplotlib.mlab import (
```

Comment:
These should probably stay but warn?

Reply:
I've just reverted the changes to pylab - if anyone tries to use the deprecated methods they'll be warned.